src/main/java/com/enderio/base/common/util/EntityCaptureUtils.java

```diff
@@ -29,16 +32,30 @@ public static List<ResourceLocation> getCapturableEntities() {
     }
 
     public enum CapturableStatus {
-        CAPTURABLE,
-        BOSS,
-        BLACKLISTED,
-        INCOMPATIBLE,
+        CAPTURABLE(Component.empty()),
+        BOSS(EIOLang.SOUL_VIAL_ERROR_BOSS),
+        BLACKLISTED(EIOLang.SOUL_VIAL_ERROR_BOSS),
```

Comment:
Making blacklisted use the boss error message could potentially be misleading? It's only minor but if a modpack blacklisted lets say pigs for example, wouldn't that display "You can't capture a boss" when right clicking a pig?

Reply:
probably an error caused by coping and pasting the Components